index.js

```diff
@@ -1,25 +1,48 @@
 'use strict';
 const path = require('path');
 const execa = require('execa');
+const filenamify = require('filenamify');
 
 const bin = path.join(__dirname, '.build/release/file-icon');
 
-module.exports = (file, size) => {
+const validate = (file, opts) => {
```

Comment:
I moved all the validation steps to a separate function so I can reuse them which makes the exported functions a lot smaller.